specs/recon/reconciliation-spec.html

```diff
@@ -0,0 +1,952 @@
+            href="https://archive.open-services.net/bin/view/Main/OSLCCoreSpecAppendixRepresentations.html">OSLC Core
+            Representations
+            Guidance</a>.</li>
+        <li class=‘conformance’>Reconciliation Providers MAY accept XML and JSON representations. Reconciliation
+          Providers
+          accepting XML or JSON SHOULD be prepared for representations that follow the guidelines
+          outlined in the <a
+            href="https://archive.open-services.net/bin/view/Main/OSLCCoreSpecAppendixRepresentations.html">OSLC Core
+            Representations Guidance</a>.</li>
+      </ul>
+
+      <p>For HTTP GET response formats for Query requests,</p>
+
+      <ul>
+        <li class=‘conformance’>Reconciliation Providers MUST provide RDF/XML and MAY provide other
+          representations such as JSON and XML.</li>
+      </ul>
+
+      <p>When Reconciliation Consumers request:</p>
+
+      <ul>
+        <li class=‘conformance’><code>application/rdf+xml</code> Reconciliation Providers MUST respond with RDF/XML
+          representation without restrictions.</li>
+        <li class=‘conformance’><code>application/xml</code> Reconciliation Providers SHOULD respond with OSLC-defined
+          abbreviated XML representation as defined in the <a
+            href="https://archive.open-services.net/bin/view/Main/OSLCCoreSpecAppendixRepresentations.html">OSLC Core
+            Representations
+            Guidance</a></li>
+      </ul>
+
+    </section>
+
+    <section id="authentication">
+      <h2>Authentication</h2>
+
+      <p>See
+        [[!OSLCCore2]] Authentication section. This specification puts no additional constraints on authentication.</p>
+    </section>
+
+    <section id="errors">
+      <h2>Error Responses</h2>
+
+      <p>See
+        [[!OSLCCore2]] Error Responses section. This specification puts no additional constraints on error responses.
+      </p>
+    </section>
+
+    <section id="pagination">
+      <h2>Pagination</h2>
+
+      <p class=‘conformance’>Reconciliation Providers SHOULD support pagination of query results and MAY
+        support pagination of a single resource’s properties as defined by the [[!OSLCCore2]].</p>
+    </section>
+
+    <section id="labels" class='informative'>
+      <h3>Labels for Relationships</h3>
+
+      <p class=‘conformance’>Relationships to other resources are represented as properties whose values are the URI of
+        the object or target
+        resource. When a relationship property is to be presented in a user interface, it may be helpful to provide an
+        informative and useful textual label for that relationship instance. (This in addition to the relationship
+        property URI and the object resource URI, which are also candidates for presentation to a user.) <a
+          href="https://archive.open-services.net/bin/view/Main/OslcCoreSpecAppendixLinks.html">OSLC Core Links
```

Comment:
1. Should be an OP Note.
2. I disagree with its content, i.e. I do not recommend to do what it recommends.

@jamsden @ndjc @graybachelor

Reply:
Can you elaborate on what I need to fix here? Is it a different element/class that I should use?

Reply:
@jam

Reply:
This should simply refer to OSLC Core 3.0 resource preview.